packages/node_modules/pouchdb-core/src/setup.js

```diff
@@ -127,4 +128,60 @@ PouchDB.fetch = function (url, opts) {
   return fetch(url, opts);
 };
 
+function ActiveTasks() {
+  this.tasks = {};
+  this.enabled = true;
+}
+
+ActiveTasks.prototype.list = function () {
+  return this.tasks;
+};
+
+ActiveTasks.prototype.add = function (task) {
```

Comment:
API naming nitpick, pairing `add` with `get` feels a bit off to me, how about `create` maybe.

I have no super strong feelings about this, or research to back this up, but `add` conveys to me “append to an existing list” which is an implementation detail from behind the API, whereas `create` conveys more “I need a new task” from the user-side of the API.

There is probably an equally strong linguistic argument for going the other way around, so the is more taste than a required change.

One thing that would be nice, but doesn’t quite fit would be to mirror the PouchDB API for doc, but that mirrors the HTTP verbs used by CouchDB and I think that is taking it a little too far (we’d use `.post()` to update.